In [2]:
import pandas as pd
import numpy as np
from scipy import stats

Level of significance is 5%

α == 0.05(95% Confidence)

Y==Discrete

X==Discrete

Since there are more than 2 variables are present, we will perform Chi-square test

# Chi-Square test

In [8]:
buyers=pd.read_csv("BuyerRatio.csv")
buyers

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [10]:
stats.chi2_contingency([buyers["East"],buyers["West"],buyers["North"],buyers["South"]])

(1.5959455386610577,
 0.6603094907091882,
 3,
 array([[  42.76531299,  442.23468701],
        [ 146.81287862, 1518.18712138],
        [ 131.11756787, 1355.88243213],
        [  72.30424052,  747.69575948]]))

P-value of Chi-Squared test == 0.66 > α

### Thus H0 is accepted.

### Buyer ratios of male-female are similar across regions.